pages/box-office/_performanceId/sell/complete.vue

```diff
@@ -51,10 +61,25 @@ export default {
   data() {
     return {
       checkedIn: false,
+      currentTime: new Date(),
     }
   },
+  computed: {
+    performanceDoorsDiffMinutes() {
+      return (
+        (Date.parse(this.booking.performance.doorsOpen) - this.currentTime) /
+        60000
+      )
+    },
+    autoCheckIn() {
+      // Performance doors are opening within 15 minutes
+      return this.performanceDoorsDiffMinutes <= 15
+    },
```

Comment:
I'd be tempted to move `autoCheckIn` from a computed variable to a method (e.g. `canAutoCheckIn()`). This would result in not needing to have both of the newly computed variables you introduced. In theory we only need to use computed variables when we are referencing them somewhere dynamically (like showing them on a page) - as we only call from another function we can make it a function.

It also avoids a potential issue caused by `currentTime` being static (it is set in the data when you load the page, so currentTime is actually just the time you opened the page. If they stay here for long enough the difference between the actual time and your `currentTime` could be long enough to cause an issue)